### PR TITLE
Cleaning up help

### LINKED ISSUE #24: [bug] orion without params
We should default to showing the available parameters. 

```
Traceback (most recent call last):
  File "/home/jtaleric/code/orion/test/bin/orion", line 8, in <module>
    sys.exit(orion())
             ^^^^^^^
  File "/home/jtaleric/code/orion/test/lib64/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jtaleric/code/orion/test/lib64/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/home/jtaleric/code/orion/test/lib64/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jtaleric/code/orion/test/lib64/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jtaleric/code/orion/test/lib64/python3.11/site-packages/orion.py", line 54, in orion
    metadata = get_metadata(test, logger)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jtaleric/code/orion/test/lib64/python3.11/site-packages/utils/orion_funcs.py", line 113, in get_metadata
    metadata=test['metadata']
             ~~~~^^^^^^^^^^^^
KeyError: 'metadata'
```

### PR DESCRIPTION
**Pull request by jtaleric**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Cleaning up the orion help output

## Related Tickets & Documents

- Related Issue #24
- Closes #24

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: orion.py [hunk 1]
```diff
@@ -28,8 +29,10 @@ def cli():
 @click.option("--debug", is_flag=True, help="log level ")
 @click.option("--hunter-analyze",is_flag=True, help="run hunter analyze")
 def orion(config, debug, output,hunter_analyze):
-    """Orion is the cli tool to detect regressions over the runs
+    """
+    Orion is the cli tool to detect regressions over the runs
 
+    \b
```
Comment (paigerube14) [inline orion.py:35]: is this needed?

Comment (jtaleric) [reply orion.py:35]: Yeah... :( 

https://click.palletsprojects.com/en/8.1.x/documentation/#preventing-rewrapping

We might be able to add empty lines in the code, but i figured the `\b` was simple enough.

Comment (paigerube14) [reply orion.py:35]: gotcha, sorry not familiar.

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Review by shashank-boyapally — APPROVED** [review]

**Pull request merged**